packages/web/src/js/element/region.ts

```diff
@@ -113,6 +112,41 @@ export const oddInfo: ElementInfo = {
     },
 };
 
+export const columnIndexerInfo: ElementInfo = {
+    getInputHandler(ref: StateRef, grid: Grid, svg: SVGSVGElement): InputHandler {
+        return getInputHandler(ref, grid, svg, '');
+    },
+    order: 42,
+    inGlobalMenu: false,
+    menu: {
+        type: 'select',
+        name: 'Column Indexer',
+        icon: 'odd-even',
+    },
+    getWarnings(value: schema.RegionElement['value'], grid: Grid, digits: IdxMap<Geometry.CELL, number>, warnings: IdxBitset<Geometry.CELL>): void {
+        const cells = idxMapToKeysArray(value || {});
+        for (const indexer of cells) {
+            const indexerValue = digits[indexer];
+            if (null == indexerValue) continue;
+            console.log(indexer, indexerValue);
+            const [ c, r ] = cellIdx2cellCoord(indexer, grid);
+            const indexee = cellCoord2CellIdx([ indexerValue - 1, r ], grid);
+            const indexeeValue = digits[indexee];
+            console.log(r, c, indexee, indexeeValue);
+            if (null == indexeeValue) continue;
```

Comment:
Can remove the console logs:
```suggestion
            if (null == indexerValue) continue;
            const [ c, r ] = cellIdx2cellCoord(indexer, grid);
            const indexee = cellCoord2CellIdx([ indexerValue - 1, r ], grid);
            const indexeeValue = digits[indexee];
            if (null == indexeeValue) continue;
```

Reply:
Grumble, grumble, grumble. Can't believe I missed that.